trendy-fashion/src/Components/Drawer.jsx

```diff
@@ -1,119 +0,0 @@
-                <Text fontSize="18px" >Men</Text>
-              </Box>
-                <FaceOutlinedIcon fontSize="medium" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Women</Text>
-              </Box>
-                <Face4OutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Kids</Text>
-              </Box>
-                <ChildCareOutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Winter Wear</Text>
-              </Box>
-                <SevereColdOutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Home & Kitchen</Text>
-              </Box>
-                <BlenderOutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Watches</Text>
-              </Box>
-                <WatchOutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Indie</Text>
-              </Box>
-                <BedOutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Jewellery</Text>
-              </Box>
-                <DiamondOutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Accessories</Text>
-              </Box>
-                <HeadphonesBatteryOutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-            <Link to="#">
-              <Flex boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px" margin="8px" ><Box>
-                <Text fontSize="18px" >Beauty</Text>
-              </Box>
-                <BrushOutlinedIcon fontSize="large" /> <Box>
-                </Box></Flex>
-            </Link>
-          </DrawerBody>
-
-          <DrawerFooter>
-
```

Comment:
Why is there nothing inside <DrawerFooter> tag? If it is an HOC component then it <DrawerFooter /> should be self-closing tag.